atlasdb-jepsen-tests/src/jepsen/atlasdb/timestamp.clj

```diff
@@ -109,5 +71,5 @@
                                      (gen/sleep 85)
                                      {:type :info, :f :stop}])))
                     (gen/time-limit 360))
-    :db (create-server)
+    :db (timelock/create-db)
```

Comment:
Note: If I'm reading this correctly we don't wait 10s and then wait until the timestamp cluster is ready before starting the timestamp test? It's right that you removed them from the general setup, but not sure if/how this affects this specific test